README.md

```diff
@@ -24,6 +24,31 @@ videojs('player_id').ready(function () {
 
       certificateUrl: '/path/to/certificate',
       licenseUrl: '/path/to/license',
+
+      // Optional settings
+
+      // Instead of a certificateUrl send the cert in a string
+      certificate: 'base64string',
+
+      // Object to add any extra license request headers that might be needed
+      licenseRequestHeaders: {
```

Comment:
can this just be `headers`? is there multiple headers that need to be set to indicate and prefix a type (`licenseRequest`)?